README.md

```diff
@@ -3,10 +3,10 @@
 The Kani Rust Verifier is a bit-precise model checker for Rust.
 
 Kani verifies:
- * memory safety -- even in unsafe Rust code
- * user-specified assertions
- * absence of panics
- * checks for some classes of undefined behavior
+ * Memory Safety -- even in unsafe Rust code
+ * User specified assertions
+ * Absence of panics
+ * Checks for some classes of undefined behavior
```

Comment:
We should also change this to "Absence of some classes of undefined behavior", since the sentence is "Kani verifies ..." which does not fit with "Check for some classes ...".

Reply:
@tedinski @adpaco-aws thoughts?

I personally agree with the suggestion.

Reply:
Actually, the way I understood this sentence is that Kani verifies the checks for those classes (here "checks" could be a noun or verb). But it may be better to rephrase just in case.

Reply:
Yeah, that could be a bit confusing to first time readers